### ISSUE TITLE
Every open console tab makes 2 requests per second back to the cluster

### ISSUE THREAD

**Issue by evilyeti** [kyma-project/console#1476]
**Description**
Console generates excessive load on the kyma backend, having 2 requests per second in idle state. With a number of users working with the cluster - this generates needless load and traffic. On laptops - it results in quicker battery drain, as this browser tab can't be suspended. On mobile connection - it burns needless data traffic. 

```
POST https://console-backend.<clustername>.<kyma domain>/graphql HTTP/1.1
Host: console-backend.<clustername>.<kyma domain>
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:71.0) Gecko/20100101 Firefox/71.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br
Referer: https://core-ui.<clustername>.<kyma domain>/namespaces
content-type: application/json
authorization: Bearer 
<token removed> 
Origin: https://core-ui.<clustername>.<kyma domain>
Content-Length: 444
Connection: keep-alive

HTTP/2.0 200 OK
access-control-allow-credentials: true
access-control-allow-origin: https://core-ui.<clustername>.<kyma domain>
content-type: application/json
vary: Origin
date: Sat, 14 Dec 2019 20:16:59 GMT
x-envoy-upstream-service-time: 20
server: istio-envoy
X-Firefox-Spdy: h2

```

```
OPTIONS https://console-backend.<clustername>.<kyma domain>/graphql HTTP/1.1
Host: console-backend.<clustername>.<kyma domain>
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:71.0) Gecko/20100101 Firefox/71.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br
Access-Control-Request-Method: POST
Access-Control-Request-Headers: authorization,content-type
Referer: https://core-ui.<clustername>.<kyma domain>/namespaces
Origin: https://core-ui.<clustername>.<kyma domain>
Connection: keep-alive

HTTP/2.0 200 OK
access-control-allow-origin: https://core-ui.<clustername>.<kyma domain>
access-control-allow-methods: GET,POST,PUT,DELETE
access-control-allow-headers: authorization,content-type
date: Sat, 14 Dec 2019 20:16:58 GMT
server: istio-envoy
X-Firefox-Spdy: h2
```

**Expected result**
No requests going back to Kyma backend when no action is triggered in the browser. Webhooks or long polls can be used as an alternative solution here to not waste network traffic and battery. 

**Actual result**
When Kyma Console is open, without anything being done in there - it makes 2 requests every second back to the graphql API:
1 x POST request
1 x OPTIONS request

**Steps to reproduce**
Open Kyma console, login to namespaces and leave the browser to sit there idle.

**Comment by kwiatekus** [comment]
Hi @evilyeti. Which  kyma version is installed on your cluster?
This problem should be fixed already with the following changes https://github.com/kyma-project/kyma/pull/6360 (available for  1.8)

**Comment by kwiatekus** [comment]
There is still auto polling configured for service catalog list view.
Please replace it with proper web-socket connection
